lib/matplotlib/backends/backend_ps.py

```diff
@@ -1219,21 +1220,26 @@ def xpdf_distill(tmpfile, eps=False, ptype='letter', bbox=None, rotated=False):
         pstoeps(tmpfile)
 
 
+@_api.deprecated("3.9")
 def get_bbox_header(lbrt, rotated=False):
     """
     Return a postscript header string for the given bbox lbrt=(l, b, r, t).
     Optionally, return rotate command.
     """
+    return _get_bbox_header(lbrt), (_get_rotate_command(lbrt) if rotated else None)
```

Comment:
Old behaviour was an empty string
```suggestion
    return _get_bbox_header(lbrt), (_get_rotate_command(lbrt) if rotated else "")
```

Reply:
oops, fixed.